spec/features/global_announcements_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Global announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    describe 'no global announcements' do
+      before do
+        instance.announcements.clear
```

Comment:
....well, I'm also confused, I'm talking about factored that line out.

Now for clearing database:
>By clearing all your state you require that all the tests be run. This removes the isolation between tests. 
I don't understand here, why clearing the database removes the isolation ?